app/src/main/java/br/com/dionataferraz/vendas/transaction/TransactionAdapter.kt

```diff
@@ -53,22 +55,38 @@ class TransactionViewHolder(
     private val listener: TransactionAdapter.Listener
 ): RecyclerView.ViewHolder(binding.root) {
 
-    fun getIconByTransactionType(type: TransactionType): Int {
-        var icon = R.drawable.ic_baseline_attach_money_24
+    fun getIconByTransactionType(type: Operation): Int {
+        var icon = R.drawable.ic_baseline_arrow_drop_down_24
 
         when {
-            TransactionType.BILL.equals(type) -> icon = R.drawable.ic_baseline_attach_money_24
-            TransactionType.LEISURE.equals(type) -> icon = R.drawable.ic_baseline_videogame_asset_24
-            TransactionType.MARKET.equals(type) -> icon = R.drawable.ic_baseline_shopping_cart_24
+            Operation.WITHDRAW.equals(type) -> icon = R.drawable.ic_baseline_arrow_drop_down_24
+            Operation.DEPOSIT.equals(type) -> icon = R.drawable.ic_baseline_arrow_drop_up_24
         }
 
         return icon
     }
 
+    @SuppressLint("SimpleDateFormat")
+    fun getFormatedDate(date: Date): String {
+        val loc = Locale("pt", "BR")
+        return SimpleDateFormat("dd MMM yyyy HH:mm", loc).format(date)
+    }
+
+    fun getFormatedName(operation: Operation): String {
+        var name = ""
+
+        when {
+            Operation.WITHDRAW.equals(operation) -> name = "Dinheiro resgatado"
+            Operation.DEPOSIT.equals(operation) -> name = "Depósito"
+        }
+
+        return name
```

Comment:
Padrão de formatação
```suggestion
    fun getFormatedName(operation: Operation): String {
        return when {
            Operation.WITHDRAW.equals(operation) -> name = "Dinheiro resgatado"
            Operation.DEPOSIT.equals(operation) -> name = "Depósito"
        }
```